### ISSUE TITLE
Support multiple branches concurrently.

### ISSUE THREAD

**Issue by Lukasa** [pyupio/pyup#216]
(Forewarning: this is a feature request. I wasn't sure where you wanted them, so I wrote this, but if this is better sent to a different place let me know and I'll happily resend to there.)

I've been using requires.io quite happily for some time to support a feature set very much like the one pyup.io provides. Recently I've started to encounter problems with being unable to batch the PRs from requires.io (see python-hyper/hyper-h2#561 for more details if you're interested), so I've become interested in moving to pyup as you do support this feature.

However, pyup does *not* appear to support being able to monitor multiple branches of the one repository. That's problematic for me, as I run multiple release branches at once and would like to keep them all up to date. It'd be fantastic if pyup was able to support this use-case. Is that likely to be possible?

**Comment by jayfk** [comment]
Yes, absolutely! Having multiple release branches is a perfectly valid use case and should be supported.

However, having **one** pull request against multiple branches all at once likely won't work. If you have 8 release branches, you'd still get 8 pull requests: one for each branch.

**Comment by Lukasa** [comment]
That would be exactly what I'd expect to see. :)

**Comment by jayfk** [comment]
All right. This will require a bit of work, I need to check how much. No ETA :).
